src-python/drbench/drbench/plotter.py

```diff
@@ -128,6 +130,35 @@ def __stream_strong_scaling_plots(self):
             style="Target",
         )
 
+    def __stream_weak_scaling_plots(self):
```

Comment:
Please add a `weak_scaling` parameter to the existing function instead of dupicating. We need the same thing for the algorithms plots below.

Reply:
I know, I commented that it is not ready for a review because I have seen your message after making this PR

Reply:
Done